scripts/ci/validate_notation_sets_contract.py

```
@@ -0,0 +1,50 @@
+    test = TEST.read_text(encoding="utf-8")
+
+    for issue in ("#320", "#309", "#182", "#175", "#126", "#304", "#196", "#177", "#208", "#274"):
+        require(source + items + mdx, issue, "required issue reference")
+
+    for nse in ("NSE-019", "NSE-020", "NSE-021"):
+        require(items, nse, "ITEMS registry entry")
+        require(mdx, nse, "public proposal index entry")
+
+    require(source, "f ; g", "generic diagrammatic composition")
+    require(source, "m \\gg= f", "Haskell Monad bind")
+    require(source, "defined only when the relevant exponential exists", "exponential existence condition")
```

Pylint (4.0.7)

[convention] Line too long (107/100)

(C0301)